src/linear_algebra/basis.lean

```diff
@@ -26,56 +26,63 @@ noncomputable theory
 
 open function lattice set submodule
 local attribute [instance] classical.prop_decidable
-variables {α : Type*} {β : Type*} {γ : Type*} {δ : Type*}
+variables {ι : Type*} {α : Type*} {β : Type*} {γ : Type*} {δ : Type*}
+          {v : ι → β}
 
 section module
 variables [ring α] [add_comm_group β] [add_comm_group γ] [add_comm_group δ]
 variables [module α β] [module α γ] [module α δ]
-variables {a b : α} {s t : set β} {x y : β}
+variables {a b : α} {x y : β}
 include α
 
-variables (α)
+section indexed
+variables {s t : set ι}
+
+variables (α) (β) (v)
```

Comment:
Is there a reason for keeping `β` explicit? It can be inferred from `v`, and I think typically `v` will be readily available.

Reply:
Done.